frontend/src/style.css

```diff
@@ -0,0 +1,339 @@
+  font-family: "Open Sans", sans-serif;
+  color: #222;
+}
+
+.logo {
+  position: fixed;
+  z-index: -3;
+  width: 100%;
+  text-align: center;
+  top: 10vh;
+  justify-content: center;
+  font-variant: small-caps;
+  font-size: 2.5em;
+  color: #eee;
+}
+
+.hidden {
+  visibility: hidden;
+}
+
+.menuButton {
+  height: 52px;
+  font-size: 1em;
+}
+
+.trigram {
+  font-size: 1.8em;
+  padding-right: 7px;
+  margin-top: -5px;
+}
+
+list-item {
+  display: block;
+}
+
+/* Used in static_pages.tsx ======================================================================= */
+
+.staticMain {
+  padding: 0.25em 1em 0.5em 1em;
+  display: flex;
+  flex-direction: column;
+}
+
+.euLogo {
+  text-align: center;
+}
+
+.euLogo img {
+  display: inline;
+  height: 15vh;
+}
+
+.euText {
+  margin: 10px 10px;
+  display: inline;
+  font-size: 0.65em;
+  text-align: left;
+}
+
+.logoImage {
+  width: 15vh;
+  max-width: 30vw;
+  height: 15vh;
+  float: right;
+}
+
+.footer {
+  z-index: -4;
+  width: 100%;
+  bottom: 0;
+  left: 0;
+  padding: 20px 40px;
+  background: white;
+  display: flex;
+  opacity: 0.9;
+}
+
+.heading {
+  margin-top: 30px;
+}
+
+.pSpace {
+  padding-bottom: 20px;
+}
+
+.pSpace p a {
+  padding-right: 4px;
+}
+
+/* desktop */
+@media screen and (min-width: 660px) {
+  .main {
+    margin-left: 20px;
+    margin-right: 20px;
+  }
+
+  .menuButton {
+    font-size: 1.2em;
+  }
+
+  .trigram {
+    font-size: 2em;
+    margin-top: -7px;
+  }
+
+  .euText {
+    font-size: 1em;
+  }
+
+  .logo {
+    font-size: 4em;
+    color: #222;
+  }
+  /* stylelint-disable */
+  body {
+    background: #ddd center / cover no-repeat url("/images/background.jpg")
+      fixed;
+    padding: 0 20px;
+  }
+  /* stylelint-enable */
+}
+
+@media screen and (max-width: 320px) {
+  .logoImage {
+    width: 10vh;
+    height: 10vh;
+  }
+
+  .euLogo img {
+    display: inline;
+    height: 10vh;
+  }
+
+  .euText {
+    font-size: 0.5em;
+  }
+}
+
+/* Used in filter_table.tsx, genebank_view.tsx and sorted_table.tsx ==================================== */
+
+.table {
+  height: 100%;
+  padding: 5px;
+  overflow-y: scroll;
+}
+
+/* Used in filter_table.tsx, sorted_table.tsx =========================================================== */
+
+.sortedTable {
+  padding: 5px;
+  overflow-y: scroll;
+}
+
+/* Used in filter_table.tsx ============================================================================= */
+
+.columnLabel {
+  padding-right: 30px;
+}
+
+.columnSelect {
+  z-index: 15;
+}
+
+.loading {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.functionLink {
+  color: blue;
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+/* Used in filter_table.tsx and sorted_table.tsx ======================================================= */
+.sorted {
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1;
+  margin: -1;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  top: 20;
+  width: 1;
+}
+
```

Comment:
There is another block above that is also "Used in filter_table.tsx and sorted_table.tsx" starting in line 197. 
I don't know if there is another system behind the order in which the classes appear here. If not, I suggest moving this part up so that `.sorted` and `.sortedTable` are after each other.

Reply:
I added comments now for each class, specifying where they are used, since I think this way is easier to edit and test them. I just found now the time to do it. This way we can, for example, remove classes that are not used, after we finish the css cleanup in subsequent PRs. 
I agree there is much more to do with css, like using name conventions, proper names, doing a cleanup of unused features, classes...Lets go for it.